Apps/Sandcastle/gallery/Particle System Weather.html

```diff
@@ -68,10 +67,8 @@
           snowParticleSize * 2.0,
           snowParticleSize * 2.0
         );
-        let snowSystem;
-
-        let snowGravityScratch = new Cesium.Cartesian3();
         const snowUpdate = function (particle, dt) {
+          let snowGravityScratch = new Cesium.Cartesian3();
```

Comment:
I like the intention of scoping the variable to the block where it's needed, but this was scoped a level up on purpose. See [the section on scratch variables in the Coding Guide](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/CodingGuide#result-parameters-and-scratch-variables).

Reply:
Nice catch 👍